frontend/src/Components/Navbar/Navbar.js

```diff
@@ -1,58 +0,0 @@
-import React from "react";
```

Comment:
Why are you delete the Navbar component??


You just need to add a media query for that not need to write the entire code again.

Reply:
I didn't delete the file, I just renamed it, because the Navbar component the Navbar function were with same name, so it was showing error.

Reply:
That entire code was written in bootstrap for js, so its hamburger wasn't working, I just used react bootstrap.

Reply:
So you can add component like navComponent

Reply:
@Castella1313 Please make above change